packages/grafana-ui/src/components/InteractiveTable/InteractiveTable.tsx

```diff
@@ -146,6 +147,10 @@ interface Props<TableData extends object> {
    * Render function for the expanded row. if not provided, the tables rows will not be expandable.
    */
   renderExpandedRow?: (row: TableData) => ReactNode;
+  /**
+   * Whether to show the "Expand all" button. Depends on renderExpandedRow to be provided. Defaults to false.
+   */
+  showExpandAll?: boolean;
```

Comment:
You don't need these two props on `BaseProps` as they're included in their own interfaces.